### PR TITLE
[zh-cn] Sync install-kubeadm.md

### PR DESCRIPTION
**Pull request by Sea-n**
There are a paragraph deleted in upstream but not synced yet.

Solve `IsSet with "invalid" (<nil>)` warning (2 of 3)
- #35813

---

~~Put comment into `{{< table >}}` shortcode, not sure it will work or not.~~

The Preview seems looks good:
https://deploy-preview-35831--kubernetes-io-main-staging.netlify.app/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm/

### CODE REVIEW

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -20,37 +20,39 @@ card:
...
 * Swap disabled. You **MUST** disable swap in order for the kubelet to work properly.
 -->
 * 一台兼容的 Linux 主机。Kubernetes 项目为基于 Debian 和 Red Hat 的 Linux
-  发行版以及一些不提供包管理器的发行版提供通用的指令
-* 每台机器 2 GB 或更多的 RAM（如果少于这个数字将会影响你应用的运行内存）
-* 2 CPU 核或更多
-* 集群中的所有机器的网络彼此均能相互连接（公网和内网都可以）
+  发行版以及一些不提供包管理器的发行版提供通用的指令。
+* 每台机器 2 GB 或更多的 RAM（如果少于这个数字将会影响你应用的运行内存）。
+* 2 CPU 核心或更多。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:46]:
```suggestion
* 2 核或更多的 CPU。
```
Similar to the previous line: `2 GB 或更多的 RAM`

**Review by windsonsea — COMMENTED** [review]
A style issue to read easily.

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md (hunk 1 above)
Comment (Sea-n) [reply content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:46]: How about "CPU 4 核心以上"?
It's weird for me to use "or more CPU", it should be "or more (CPU) cores".

Comment (Sea-n) [reply content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:46]: (I know it's double standards, but "or more RAM" is acceptable for me.)

Comment (windsonsea) [reply content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:46]: CPU 2 核心及以上

yes, it's all right.

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 2]
```diff
@@ -20,37 +20,39 @@ card:
...
-<img src="https://raw.githubusercontent.com/kubernetes/kubeadm/master/logos/stacked/color/kubeadm-stacked-color.png" align="right" width="150px">This page shows how to install the `kubeadm` toolbox.
+<img src="/images/kubeadm-stacked-color.png" align="right" width="150px"></img>
+This page shows how to install the `kubeadm` toolbox.
 For information on how to create a cluster with kubeadm once you have performed this installation process, see the [Creating a cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) page.
 -->
-<img src="/images/kubeadm-stacked-color.png" align="right" width="150px">本页面显示如何安装 `kubeadm` 工具箱。
+<img src="/images/kubeadm-stacked-color.png" align="right" width="150px"></img>
+本页面显示如何安装 `kubeadm` 工具箱。
 有关在执行此安装过程后如何使用 kubeadm 创建集群的信息，请参见
 [使用 kubeadm 创建集群](/zh-cn/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)页面。
```
Comment (tengqm) [inline content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:30]:
```suggestion
有关在执行此安装过程后如何使用 kubeadm 创建集群的信息，
请参见[使用 kubeadm 创建集群](/zh-cn/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)。
```

**Review by tengqm — COMMENTED** [review]
/approve
Found a nit, not directly related.

**Pull request merged**